invoice/src/test/java/org/killbill/billing/invoice/api/user/TestDefaultInvoiceUserApi.java

```diff
@@ -701,4 +702,42 @@ public void testCreditWithoutDraftInvoice() throws Exception {
         Assert.assertEquals(updatedExternalChargeInvoice.getChargedAmount().compareTo(new BigDecimal(300)), 0);
 
     }
+    
+    @Test(groups = "slow")
+    public void testRetrieveInvoicesByAccount() throws Exception {
+
+        final Account account = invoiceUtil.createAccount(callContext);
+        final UUID accountId = account.getId();
+
+        // Create invoice1
+        final InvoiceItem externalCharge1 = new ExternalChargeInvoiceItem(null, accountId, null, UUID.randomUUID().toString(), clock.getUTCToday(), null, new BigDecimal(300), accountCurrency, null);
+        invoiceUserApi.insertExternalCharges(accountId, clock.getUTCToday(), List.of(externalCharge1), true, null, callContext).get(0);
```

Comment:
What's the `get(0)` (same question below)?

Reply:
It's not required, deleted it.